packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -1486,3 +1490,48 @@ describe('range operations', () => {
     testIsRangeOperation('~', false);
   });
 });
+
+describe('Sorting', () => {
+  function mockSort(index: number): Sort {
+    return ({
+      column: { index, name: `name_${index}` },
+    } as unknown) as Sort;
+  }
+
+  const sortList = {
+    empty: [] as Sort[],
+    hasThree: [mockSort(333), mockSort(444), mockSort(999)],
+  };
+
+  describe('getSortIndex', () => {
+    // sort, columnName, expected
+    const testCases: [Sort[], ColumnName, number | null][] = [
+      [sortList.empty, 'name_999', null],
+      [sortList.hasThree, 'non-existing', null],
+      [sortList.hasThree, 'name_999', 2],
+    ];
+
+    it.each(testCases)(
```

Comment:
TIL. Didn't know there was a `it.each`. Could have used this in a bunch of other cases.

Reply:
@mofojed There's also a string literal syntax supported by `it.each` that can be used to inject test variables. I have rarely used it but worth checking out the docs. https://jestjs.io/docs/api#2-testeachtablename-fn-timeout

Reply:
That's pretty neat! Good to know.